app/views/layouts/_google_analytics.html.erb

```diff
@@ -1,11 +1,11 @@
-<% if Rails.env.production? and Settings::GoogleAnalytics.send_google_analytics == true and !Settings::GoogleAnalytics.google_analytics_code.blank?  %>
+<% if Rails.env.production? and Settings::GoogleAnalytics.send_google_analytics == true and !Rails.application.secrets[:google_analytics_code].blank?  %>
```

Comment:
I was thinking... "why didn't I catch this locally"... `if Rails.env.production?` 🤦🏼 😂